lighthouse-core/gather/gatherers/seo/tap-targets.js

```diff
@@ -283,6 +283,7 @@ function gatherTapTargets() {
   for (const {tapTargetElement, visibleClientRects} of tapTargetsWithVisibleClientRects) {
     targets.push({
       clientRects: visibleClientRects,
+      boundingRect: JSON.parse(JSON.stringify(tapTargetElement.getBoundingClientRect())),
```

Comment:
make a note of why we do this? I'm assuming it's because protocol will serialize to empty object but `JSON.stringify` will do the right thing, but I don't know :)

Reply:
thats correct, no idea why protocol does that

Reply:
because getters